### PR TITLE
[Core] Remaining clean up of python scripts 

### PR DESCRIPTION
**Pull request by loumalouomega**
Remaining from #3453

### CODE REVIEW

File: kratos/python_scripts/assign_vector_variable_to_entities_process.py [hunk 1]
```diff
@@ -65,41 +63,18 @@ def __init__(self, Model, settings ):
...
-            z_params.AddValue("model_part_name",settings["model_part_name"])
-            z_params.AddValue("mesh_id",settings["mesh_id"])
-            z_params.AddValue("interval",settings["interval"])
-            z_params.AddValue("value",settings["value"][2])
-            z_params.AddEmptyValue("variable_name").SetString(settings["variable_name"].GetString() + "_Z")
-            z_params.AddValue("local_axes",settings["local_axes"])
-            z_params.AddValue("entities",settings["entities"])
-            self.aux_processes.append( assign_scalar_variable_to_entities_process.AssignScalarVariableToEntitiesProcess(Model, z_params) )
+        list_directions = ["_X", "_Y", "_Z"]
+        for i_dir in range(3):
```
Comment (philbucher) [inline kratos/python_scripts/assign_vector_variable_to_entities_process.py]:
```suggestion
        for i_dir, dir in enumerate(["_X", "_Y", "_Z"]):
```

this is basically the only review comment.
can you please check if this is also applicable for the other processes?

**Review by philbucher — COMMENTED** [review]
minor comment

**Comment by philbucher** [comment]
some tests in fluid & structure are crashing

**Comment by philbucher** [comment]
seems to be related to this PR, others are working with you recent big merge

**Comment by loumalouomega** [comment]
Checking in out

**Review by philbucher — APPROVED** [review]
looks good, thx

**Pull request merged**